src/ui/qgisapp.ui

```diff
@@ -1017,7 +1017,10 @@
      <normaloff>:/images/themes/default/mActionVertexTool.svg</normaloff>:/images/themes/default/mActionVertexTool.svg</iconset>
    </property>
    <property name="text">
-    <string>Vertex Tool</string>
+    <string>&amp;Vertex Tool on All Layers</string>
```

Comment:
I think that except the first, the words should be lower case.

Reply:
I don't have a strong opinion but it seems that all other names are worded in this manner (at least in `src/ui/qgisapp.ui`):

```
...
<string>New 3D Map View</string>
...
<string>New Report</string>
...
<string>Paste Layer/Group</string>
...
```

@3nids What do you think?

Reply:
@DelazJ What do you think?

Reply:
HIG: https://docs.qgis.org/testing/en/docs/developers_guide/hig.html

Reply:
I'd avoid the issue and go "Vertex Tool (All Layers)" and "Vertex Tool (Current Layer)"